docs/source/contribution.rst

```diff
@@ -3,214 +3,77 @@
-* **NoCompat**: disrupts backward compatibility
-* **Test**: test fixes and updates
-* **Document**: documentation fixes and improvements
-* **Example**: fixes and improvements on the examples
-* **Install**: fixes installation script
-* **Contribution-Welcome**: issues that we request for contribution (only issues are categorized to this)
-* **Other**: other issues and PRs
-
-Multiple tags might be labeled to one issue/PR.
-**Note that revision releases cannot include PRs in Feature and NoCompat categories.**
+First steps in contributing to Chainer often involve filing an issue or creating a PR.
+This section describes how to do so.
 
 How to File an Issue
 ~~~~~~~~~~~~~~~~~~~~
 
-On registering an issue, write precise explanations on how you want Chainer to be.
-Bug reports must include necessary and sufficient conditions to reproduce the bugs.
-Feature requests must include **what** you want to do (and **why** you want to do, if needed) with Chainer.
-You can contain your thoughts on **how** to realize it into the feature requests, though **what** part is most important for discussions.
+To file an issue on GitHub, you often only need to follow instructions given by the template.
+Write precise explanations on how you want Chainer to behave or include necessary and sufficient conditions to reproduce the bugs.
+Feature requests should include **what** you want to do and preferably **why**.
+You may additionally suggest **how**.
 
 .. warning::
 
-   If you have a question on usages of Chainer, it is highly recommended to send a post to `StackOverflow <https://stackoverflow.com/>`_ or `Chainer User Group <https://groups.google.com/forum/#!forum/chainer>`_ instead of the issue tracker.
+   If you have a question regarding the usage of Chainer, it is recommended to send a post to `StackOverflow <https://stackoverflow.com/>`_ or the `Chainer User Group <https://groups.google.com/forum/#!forum/chainer>`_ instead of the issue tracker.
    The issue tracker is not a place to share knowledge on practices.
-   We may suggest these places and immediately close how-to question issues.
 
 How to Send a Pull Request
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you can write code to fix an issue, we encourage to send a PR.
-
-First of all, before starting to write any code, do not forget to confirm the following points.
+If you can write code to fix an issue, it is encouraged to send a PR.
 
-- Read through the :ref:`coding-guide` and :ref:`testing-guide`.
-- Check the appropriate branch that you should send the PR following :ref:`contrib-git-branches`.
-  If you do not have any idea about selecting a branch, please choose the ``master`` branch.
+In that case, confirm the following points before starting to write any code.
 
-In particular, **check the branch before writing any code.**
-The current source tree of the chosen branch is the starting point of your change.
+- Read the :ref:`coding-guide` and the :ref:`testing-guide`.
```

Comment:
```suggestion
- Read :ref:`coding-guide` and :ref:`testing-guide`.
```
Looking at the rendered document, I feel it's more natural as a hyperlinked document to show the linked section names without articles here.